addons/l10n_fi/static/description/finland.svg

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we use svg elsewhere?  Not even sure it is still supported.  Maybe just remove this file.

Reply:
It works

Reply:
Mmmh, no, that was the png next to it.

Reply:
I removed the svg

Reply:
But normally, elsewhere it is the png that is used anyways. (I guess)

Reply:
Indeed, I removed the svg and only kept the png